square/shwap/row_namespace_data.go

```diff
@@ -175,16 +177,16 @@ func (rnd RowNamespaceData) Verify(roots *share.AxisRoots, namespace share.Names
 func (rnd RowNamespaceData) verifyInclusion(rowRoot []byte, namespace share.Namespace) bool {
 	leaves := make([][]byte, 0, len(rnd.Shares))
 	for _, sh := range rnd.Shares {
-		namespaceBytes := share.GetNamespace(sh)
-		leave := make([]byte, len(sh)+len(namespaceBytes))
+		namespaceBytes := sh.Namespace().Bytes()
+		leave := make([]byte, len(sh.ToBytes())+len(namespaceBytes))
 		copy(leave, namespaceBytes)
-		copy(leave[len(namespaceBytes):], sh)
+		copy(leave[len(namespaceBytes):], sh.ToBytes()) // rework
```

Comment:
This PR ?

Reply:
it is already reworked. Forgot to remove the comment